### ISSUE TITLE
iOS subscribe: Error Domain=CBATTErrorDomain Code=3 "Writing is not permitted"

### ISSUE THREAD

**Issue by ragcsalo** [randdusing/cordova-plugin-bluetoothle#475]
When I try to subscribe for notifications on a characteristic, I get this error:

`Error Domain=CBATTErrorDomain Code=3 "Writing is not permitted"`

I don't want to write anything, just be notified when the characteristic value changes...
The plugin description says "**The Client Configuration descriptor will automatically be written to enable notification/indication based on the characteristic's properties.**" Maybe that's the problem? There is no Write access to this characteristic... so why does the plugin try to write something automatically??

**Comment by hannibaldinski** [comment]
Does the characteristic allow for notification/subscriptions?

**Comment by ragcsalo** [comment]
Yes of course. :-) As a matter of fact, only "Notify" is available for this characteristic... I don't know why I get a "Writing is not permitted" error when subscribing. Maybe this is an iOS specific error, I also couldn't subscribe by using another BT LE plugin. Also 3rd party apps were unable to subscribe... On Android, 3rd party apps could successfully subscribe. (I also tried this plugin on Android, but I stuck at the scanning process with error "Application registration failed". But that's another issue...)

- Device: Xiaomi Mi Band 2
- Service: FEE0
- Characteristic: 00000010-0000-3512-2118-0009AF100700

**Comment by gregor-srdic** [comment]
@ragcsalo did you find a solution for this?
